web/src/main/webapp/js/translate.js

```diff
@@ -6,11 +6,11 @@ var mathTools = require('./tools/math.js');
 
 function tr2(key, args) {
     if (key === null) {
-        log("ERROR: key was null?");
+        console.log("ERROR: key was null?");
         return "";
     }
     if (defaultTranslationMap === null) {
-        log("ERROR: defaultTranslationMap was not initialized?");
```

Comment:
Thanks, it looks like `log(` is no longer defined (?)

We should then fix the several other places too (maybe another issue)